Modules/Core/Common/include/itkImageTransformer.hxx

```diff
@@ -351,6 +351,21 @@ ImageTransformer< TInputImage >
   if ( threadId < total )
     {
     str->Filter->ThreadedGenerateData(splitRegion, threadId);
+#if defined( ITKV4_COMPATIBILITY )
+    if ( str->Filter->GetAbortGenerateData() )
+      {
+      std::string msg;
+      ProcessAborted e( __FILE__, __LINE__ );
+      msg += "Object " + std::string( str->Filter->GetNameOfClass() ) + ": AbortGenerateData was set!";
+      e.SetDescription( msg );
+      throw e;
+      }
+    else if ( !str->Filter->GetDynamicMultiThreading() // progress reporting is not done in MultiThreaders
```

Comment:
In file included from /home/blowekamp/src/ITK/Modules/Core/Common/include/itkImageTransformer.h:251:0,
                 from /home/blowekamp/src/ITK/Modules/Numerics/Statistics/include/itkImageToHistogramFilter.h:24,
                 from /home/blowekamp/src/ITK/Modules/Filtering/Thresholding/include/itkHistogramThresholdImageFilter.hxx:22,
                 from /home/blowekamp/src/ITK/Modules/Filtering/Thresholding/include/itkHistogramThresholdImageFilter.h:213,
                 from /home/blowekamp/src/ITK/Modules/Filtering/Thresholding/include/itkOtsuThresholdImageFilter.h:22,
                 from /home/blowekamp/src/ITK/Modules/Filtering/BiasCorrection/test/itkN4BiasFieldCorrectionImageFilterTest.cxx:22:
/home/blowekamp/src/ITK/Modules/Core/Common/include/itkImageTransformer.hxx: In instantiation of ‘static void* itk::ImageTransformer<TInputImage>::ThreaderCallback(void*) [with TInputImage = itk::Image<float, 2u>; itk::ITK_THREAD_RETURN_FUNCTION_CALL_CONVENTION = void*]’:
/home/blowekamp/src/ITK/Modules/Core/Common/include/itkImageTransformer.hxx:302:3:   required from ‘void itk::ImageTransformer<TInputImage>::GenerateData() [with TInputImage = itk::Image<float, 2u>]’
/home/blowekamp/src/ITK/Modules/Filtering/BiasCorrection/test/itkN4BiasFieldCorrectionImageFilterTest.cxx:353:1:   required from here
/home/blowekamp/src/ITK/Modules/Core/Common/include/itkImageTransformer.hxx:363:15: error: ‘using ObjectType = class itk::ImageTransformer<itk::Image<float, 2u> >’ has no member named ‘GetDynamicMultiThreading’
     else if ( !str->Filter->GetDynamicMultiThreading() // progress reporting is not done in MultiThreaders
               ^
/home/blowekamp/src/ITK/Modules/Core/Common/include/itkImageTransformer.hxx: In instantiation of ‘static void* itk::ImageTransformer<TInputImage>::ThreaderCallback(void*) [with TInputImage = itk::Image<float, 3u>; itk::ITK_THREAD_RETURN_FUNCTION_CALL_CONVENTION = void*]’:
/home/blowekamp/src/ITK/Modules/Core/Common/include/itkImageTransformer.hxx:302:3:   required from ‘void itk::ImageTransformer<TInputImage>::GenerateData() [with TInputImage = itk::Image<float, 3u>]’
/home/blowekamp/src/ITK/Modules/Filtering/BiasCorrection/test/itkN4BiasFieldCorrectionImageFilterTest.cxx:353:1:   required from here
/home/blowekamp/src/ITK/Modules/Core/Common/include/itkImageTransformer.hxx:363:15: error: ‘using ObjectType = class itk::ImageTransformer<itk::Image<float, 3u> >’ has no member named ‘GetDynamicMultiThreading’
[637/2860] Building CXX object Modules/Core/TestKernel/src/CMakeFiles/ITKTestKernel.dir/itkTestDriverInclude.cxx.o

Reply:
Let's see if the new amend fixes it.